docs/overview/contributing/building_the_docs.rst

```diff
@@ -5,6 +5,16 @@ This document describes how to build the Ivy docs. If you want to know more abou
 our custom building pipeline work, check our `Building the Docs Pipeline
 <../deep_dive/building_the_docs_pipeline.rst>`_ deep dive
 
+.. note::
+
+    The doc-builder was developed originally for linux, although in theory you can run
+    it on any platform (supporting either docker or windows), we only tested it on
+    linux.
+
+    We recommended the use of convenience script if you are planning to regularly build
+    the docs, as it won't redownload the dependencies every time. Also, the use of
+    GitHub codespaces will help you with faster network if your internet is slow.
```

Comment:
We should avoid using passive voice and talk with the contributor directly. 

```suggestion
    Recommendation: 
    You can use the convenience script if you build the docs regularly,
    as it will not re-download the dependencies. 
    
    If you have a slow internet connection, you can use GitHub Codespaces since it will help you to build the 
    docs faster since our script downloads large dependency files.
```